src/OrchardCore.Modules/OrchardCore.Users/Migrations.cs

```diff
@@ -240,5 +243,50 @@ public int UpdateFrom11()
 
             return 12;
         }
+
+        public int UpdateFrom12()
+        {
+            ShellScope.AddDeferredTask(async scope =>
+            {
+                var session = scope.ServiceProvider.GetRequiredService<ISession>();
+                var dbConnectionAccessor = scope.ServiceProvider.GetService<IDbConnectionAccessor>();
+                var logger = scope.ServiceProvider.GetService<ILogger<Migrations>>();
+                var tablePrefix = session.Store.Configuration.TablePrefix;
+                var documentTableName = session.Store.Configuration.TableNameConvention.GetDocumentTable();
+                var table = $"{session.Store.Configuration.TablePrefix}{documentTableName}";
+
+                using var connection = dbConnectionAccessor.CreateConnection();
+                await connection.OpenAsync();
+
+                using var transaction = connection.BeginTransaction(session.Store.Configuration.IsolationLevel);
+                var dialect = session.Store.Configuration.SqlDialect;
+
+                try
+                {
+                    logger.LogDebug("Updating User Settings");
+
+                    var quotedTableName = dialect.QuoteForTableName(table, session.Store.Configuration.Schema);
+                    var quotedContentColumnName = dialect.QuoteForColumnName("Content");
+                    var quotedTypeColumnName = dialect.QuoteForColumnName("Type");
+
+                    var updateCmd = $"UPDATE {quotedTableName} " +
```

Comment:
It's ok to use a single line, and if you are adventurous you can even try the new `"""` raw string syntax, might be the best option even here. https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/raw-string

Reply:
But it will probably not work with templated strings :/

Reply:
`"""` is not available in C# 10.

I created a mile long SQL command :) any other suggestions?